android/src/main/kotlin/fr/dooz/nordic_nrf_mesh/DoozMeshNetwork.kt

```diff
@@ -134,6 +134,12 @@ class DoozMeshNetwork(private val binaryMessenger: BinaryMessenger, var meshNetw
                 }
                 result.success(nodes)
             }
+            "getProvisionersUUID" -> {
+                val provisionersUUID = meshNetwork.provisioners.map { provisioner ->
+                    provisioner.provisionerUuid
```

Comment:
What's the type returned by this getter ?

Reply:
it will return List <String>

Reply:
a list of what ?
what is returned by `provisioner.provisionerUuid` ?

Reply:
it was list of String.